dspot/src/main/java/eu/stamp_project/dspot/selector/JacocoCoverageSelector.java

```diff
@@ -222,6 +223,12 @@ public TestSelectorElementReport report() {
                     .append("%")
                     .append(AmplificationHelper.LINE_SEPARATOR);
             lastReport = new TestSelectorElementReportImpl(report.toString(), jsonReport(coverageResults));
+
+            // Report to mongodb
+            MongodbManager.getInstance().reportJacocotInformation(this.currentClassTestToBeAmplified.getQualifiedName(),
```

Comment:
global state + mongo hardcoded to be avoided. All calls to `MongodbManager.getInstance()` should be replaced `config.getInformationCollector()`